pp_ctl.c

```diff
@@ -4762,27 +4761,23 @@ PP(pp_entereval)
 
 PP(pp_leaveeval)
 {
+    dSP;
     SV **oldsp;
     U8 gimme;
     PERL_CONTEXT *cx;
     OP *retop;
-    int failed;
+    int failed = 0;
     CV *evalcv;
     bool keep;
-
     PERL_ASYNC_CHECK();
+    int module_true = 0;
```

Comment:
Since we're using C99 now don't forget this can be `bool` - we're only ever using it in a truthy capacity so it's better to be `bool` and assign `true` or `false`